modules/installing-ocp-agent.adoc

```diff
@@ -0,0 +1,159 @@
+
+. Navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-{product-version}/[OpenShift mirror site] and download the latest version of the tarball that matches your operating system.
+
+. Extract the binary.
++
+[source,terminal]
+----
+$ tar xvf <file> -C ~/.local/bin
+----
++
+The file contains the Agent-based installer.
+
+. Install `nmstate` dependency.
++
+[source,terminal]
+----
+$ sudo dnf install /usr/bin/nmstatectl -y
+----
+
+. Place the `openshift-install` binary in a directory that is on your PATH.
+
+. Create a directory to store the cluster manifests.
++
+[source,terminal]
+----
+$ mkdir ~/<directory_name>
+----
+
+. Create the `install-config.yaml` file.
++
+[source,yaml]
+----
+cat << EOF > ./cluster-manifests/install-config.yaml
+apiVersion: v1
+baseDomain: test.example.com
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+  name: worker
+  replicas: 0
+controlPlane:
+  architecture: amd64
+  hyperthreading: Enabled
+  name: master
+  replicas: 1
+metadata:
+  name: sno-cluster <1>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 192.168.111.0/16
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  none: {}
+pullSecret: '<pull_secret>' <2>
+sshKey: |
+  '<ssh_pub_key>' <3>
+  EOF
+----
++
+<1> Required.
+<2> Enter your pull secret.
+<3> Enter your ssh public key.
+
+. Create the `agent-config.yaml` file.
++
+[source,yaml]
+----
+  cat > agent-config.yaml << EOF
+  kind: AgentConfig
+  metadata:
+    name: sno-cluster
+  rendezvousIP: 192.168.111.80 <1>
+  hosts:
+    - hostname: master-0
+      interfaces:
+        - name: eno1
+          macAddress: 00:ef:44:21:e6:a5
+      networkConfig:
+        interfaces:
+          - name: eno1
+            type: ethernet
+            state: up
+            mac-address: 00:ef:44:21:e6:a5
+            ipv4:
+              enabled: true
+              address:
+                - ip: 192.168.111.80
+                  prefix-length: 23
+              dhcp: false
+        dns-resolver:
+          config:
+            server:
+              - 192.168.111.1
+        routes:
+          config:
+            - destination: 0.0.0.0/0
+              next-hop-address: 192.168.111.2
+              next-hop-interface: eno1
+              table-id: 254
+  EOF
+----
++
+<1> Specify the IP address.
+
++
+. Create the Agent image.
++
+[NOTE]
+====
+Ensure that the image is created along with the **auth** folder. You can verify this with the following command:
+
+[source,terminal]
+----
+$ ls
+agent.iso  auth
+----
+====
++
+[source,terminal]
+----
+$ ./openshift-install agent create image
+----
+
+. Boot the `agent.iso` image on the bare metal machines. You can run the image on any Linux distribution.
+
+. To track the progress and verify sucessful installation, run the following command:
++
+[source,terminal]
+----
+$ ./openshift-install agent wait-for install-complete
```

Comment:
```suggestion
$ openshift-install agent wait-for install-complete
```
Since we put "openshift-install" in "/home/user/.local/bin", which should be in the path, the executable will be found normally, no need for a leading "./"